pyanaconda/modules/payloads/source/source_base.py

```diff
@@ -84,3 +91,43 @@ def tear_down_with_tasks(self):
         :rtype: list[task]
         """
         pass
+
+
+class MountingSourceBase(PayloadSourceBase, ABC):
+    """Base class for sources that use mounting.
+
+    Implements some common functionality, most notably generation of mount point paths.
+    """
+    # pylint: disable=abstract-method
+
+    def __init__(self):
+        super().__init__()
+        self._mount_point = MountPointGenerator.generate_mount_point(self.type.value.lower())
+
+    def is_ready(self):
+        """This source is ready for the installation to start.
+
+        :return: ready or not
+        :rtype: bool
+        """
+        res = os.path.ismount(self._mount_point)
+        log.debug("Source is set to %s ready state", res)
```

Comment:
@jkonecny12, isn't this message a leftover from debugging? I would expect the payload to log this kind of messages when it has sources in invalid state. Otherwise, it is not really useful.

Reply:
Removed.